servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NoopTransportObserver.java

```diff
@@ -71,6 +72,11 @@ public void onFlush() {
         public void onTransportHandshakeComplete() {
         }
 
+        @Override
```

Comment:
Maybe for a follow-up pr: but why do we have NoopTransportObservers in transport internal and transport api? Maybe we can make the internal one use the one in API?

Reply:
The original motivation was to avoid null-checks in our internal codebase, but overtime we found it not user-friendly to always require to return an observer. Users struggle to understand what they suppose to return when they are not interested in callbacks of sub-observer. Even if we make it public in transport-api, it's still hard to discover and easy to return your own, which will not give expected reduction of the overhead. The long-term plan is to make all callbacks that return another observer `@Nullable`.